example/tutorials/code/rtt-matmuladd.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+
+# Import rosetta package
+import latticex.rosetta as rtt
+import tensorflow as tf
+
+# You can activate a backend protocol, here use SecureNN
+rtt.py_protocol_handler.set_loglevel(0);
+rtt.activate("SecureNN")
+
+
+# Get private data from Alice (input x), Bob (input y)
+x0 = tf.Variable(rtt.private_input(0, [[1, 0], [0, 1]]))
```

Comment:
More test cases are needed, such ad one side of the operands  may be constant value.